code/unclassified/fibonacci.cpp

```diff
@@ -0,0 +1,22 @@
+#include <bits/stdc++.h>

```

Comment:
This code must be in [this code](https://github.com/OpenGenus/cosmos/tree/master/code/mathematical-algorithms/fibonacci_number)

Kindly make corresponding change 👍